tutorials/using-cloud-vpn-with-cisco-asr/index.md

```diff
@@ -0,0 +1,744 @@
+---
+title: How to Set Up VPN between Cisco ASR and Cloud VPN
+description: Learn how to build site-to-site IPSEC VPN between Cisco ASR and Cloud VPN.
+author: ashishverma
+tags: Compute Engine, Cloud VPN, Cisco ASR
+date_published: 2017-08-25
+---
+
+![ASR_Family](cisco_asr_family.jpeg)
+
+*Courtesy of Cisco Systems, Inc. Unauthorized use not permitted. Cisco® is a registered trademark or trademark of Cisco Systems, Inc. and/or its affiliates in the United States and certain other countries.*
```

Comment:
It came from the standard template used earlier for all interop guides. I feel like removing this image altogether to avoid any copyright related issues.